default-config.json

```diff
@@ -82,5 +64,33 @@
 			},
 			"requireAuth": "auto"
 		}
+	},
+	"public": {
+		"default": {
+		"commerce-core-endpoint": "https://www.aemshop.net/graphql",
+		"commerce-endpoint": "https://www.aemshop.net/cs-graphql",
+		"headers": {
+			"cs": {
+			"Magento-Customer-Group": "b6589fc6ab0dc82cf12099d1c2d40ab994e8410c",
+			"Magento-Store-Code": "main_website_store",
+			"Magento-Store-View-Code": "default",
+			"Magento-Website-Code": "base",
+			"x-api-key": "4dfa19c9fe6f4cccade55cc5b3da94f7",
+			"Magento-Environment-Id": "f38a0de0-764b-41fa-bd2c-5bc2f3c7b39a"
+			}
+		},
+		"analytics": {
+			"base-currency-code": "USD",
+			"environment": "Production",
+			"store-id": 1,
+			"store-name": "Main Website Store",
+			"store-url": "https://www.aemshop.net",
+			"store-view-id": 1,
+			"store-view-name": "Default Store View",
+			"website-id": 1,
+			"website-name": "Main Website"
+		}
+		}
+	
```

Comment:
1. Move this public config to a new default-publicConfig.yaml.
2. Replace all hardcoded values with variables, such as `{ORG}`, `{REPO}` etc
3. Add info about how to set this to the README (upload site config first, then upload public config)
4. Add info to README about using content config file(s) instead of publicConfig.